src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -25,6 +25,16 @@ define(["dojo/_base/declare", "framework/PluginBase"],
 
                 // Hide the print button until the identify feature has been used.
                 $(this.printButton).hide();
+
+                // Display a clickable image thumbnail
+                $(this.container)
+                    .append('<p>Click the thumbnail to see a sample modal image popup</p>')
+                    .append('<div class="identify-point-plugin-thumbnail-image"' +
```

Comment:
Sounds good. I'll drop the `div` and move the cursor style and listener to the `img`.